cli.ts

```diff
@@ -127,51 +130,140 @@ const rootDir = getProjectRoot(process.cwd());
+    describe: "Creates a new hathora project",
+    handler: (_argv) => {
+      if (existsSync(join(serverDir, "impl.ts"))) {
+        console.error(
+          `${chalk.red("Cannot init inside existing project, delete ")}` +
+            `${chalk.blue.underline("impl.ts")}` +
+            `${chalk.red(" to regenerate")}`
+        );
+      } else {
+        generate(rootDir, "templates/bootstrap");
+        generateLocal();
+      }
+    },
+  })
+  .command({
+    command: "start",
+    aliases: ["up", "s"],
+    describe: "Starts the hathora server",
+    handler: (_argv) => {
+      startServer().then(startFrontends);
+    },
+  })
+  .command({
+    command: "dev",
+    aliases: ["development", "d"],
+    describe: "Starts the server in development mode",
+    handler: (_argv) => {
+      if (!existsSync(join(serverDir, "impl.ts"))) {
+        console.error(
+          `${chalk.red("Missing impl.ts, make sure to run")}` +
+            `${chalk.blue.bold(" hathora init ")}` +
+            `${chalk.red("first")}`
+        );
+      } else {
+        generateLocal();
+      }
+      install();
+      startServer().then(startFrontends);
+    },
+  })
+  .command({
+    command: "save [stateId] [saveName]",
+    aliases: ["gamesave", "sv"],
+    describe: "Creates a named save game from a specific state id",
+    handler: (argv) => {
+      copySync(join(rootDir, "data", argv.stateId as string), join(rootDir, "data", "saves", argv.saveName as string));
+    },
+  })
+  .command({
+    command: "build",
+    aliases: ["b"],
+    describe: "Builds the project",
+    handler: (_argv) => {
+      if (!existsSync(join(serverDir, "impl.ts"))) {
+        console.error(
+          `${chalk.red("Missing impl.ts, make sure to run")}` +
+            `${chalk.blue.bold(" hathora init ")}` +
+            `${chalk.red("first")}`
+        );
+      } else {
+        generate(rootDir, "templates/base", getAppConfig());
+      }
+      install();
+      build();
+    },
+  })
+  .command({
+    command: "install",
+    aliases: ["i"],
+    describe: "Install hathora dependencies",
+    handler: (_argv) => {
+      install();
+    },
+  })
+  .command({
+    // For backwards compatibility
+    command: "create-plugin-native <type>",
+    describe: "Creates a native plugin",
+    handler: (argv) => {
+      generate(rootDir, "templates/plugin/native", {
+        val: argv.type as string,
+      });
+    },
+  })
+  .command({
+    // For backwards compatibility
+    command: "create-plugin-react <type>",
+    describe: "Creates a react plugin",
+    handler: (argv) => {
+      generate(rootDir, "templates/plugin/react", { val: argv.type as string });
+    },
+  })
+  .command({
+    // For backwards compatibility
+    command: "create-plugin-view <type>",
+    aliases: ["create-plugin-lit"],
+    describe: "Creates a lit plugin",
+    handler: (argv) => {
+      generate(rootDir, "templates/plugin/lit", { val: argv.type as string });
+    },
+  })
```

Comment:
we don't need to maintain compatibility for these commands at this stage, just need to update the docs with the new version of the command

Reply:
I've kept them in, updated the docs, and marked them as deprecated. I'd suggest we remove them in a future version.

We should add some validation or error handling around what happens if someone inputs something other than `react`, `lit`, and `native`. Right now they'll get a missing directory error I think.